app/src/main/java/org/openobservatory/ooniprobe/activity/RunningActivity.java

```diff
@@ -154,13 +149,15 @@ private void applyUIChanges(){
             name.setText(service.task.currentTest.getName());
         else
             name.setText(getString(service.task.currentTest.getLabelResId()));
-        runtime = service.task.currentSuite.getRuntime(preferenceManager);
         getWindow().setBackgroundDrawableResource(service.task.currentSuite.getColor());
         if (Build.VERSION.SDK_INT >= 21) {
             getWindow().setStatusBarColor(service.task.currentSuite.getColor());
         }
         animation.setAnimation(service.task.currentSuite.getAnim());
-        progress.setMax(service.task.currentSuite.getTestList(preferenceManager).length * 100);
+        progress.setMax((int)Stats.of(Lists.transform(
+                service.task.testSuites,
+                input -> input.getTestList(preferenceManager).length * 100
```

Comment:
I would suggest renaming the argument for the lambda to `ts` to indicate that it's a testSuite. 

Initially I was confused because I thought we were counting inputs here, but we are actually counting test names. That is to say that when you call `getTestList().length` on the `Websites` testSuite you will get 1, while when you call the same on InstantMessagging you will get 4.

Question: does this have an impact on the progress bar moving slower than it should on the websites test group?